### ISSUE TITLE
Multiple Instances

### ISSUE THREAD

**Issue by ayanm00** [rudders/homebridge-platform-wemo#129]
Hi,

So all of the devices in my house are still running iOS 10.3.3 because of how unpolished iOS 11 has been. Unfortunately, Apple disabled the functionality of sharing homes on devices running a version of iOS older than 11.2.1. What I ended up doing was running multiple instances of Homebridge for each Apple ID that would be controlling devices in my home (yeah, lots of work). What I have noticed is that this plugin only updates the status of one of those instances when a light is turned on or off. Is there a way to have all the instances detect when a light is turned on or off?

Thanks!

**Comment by DMBlakeley** [comment]
I have been running Homebridge with iOS11.2.x since first available.  Yes, a bit of a rough start but I have had no issues with 11.2.5 and Homebridge/HomeKit.  By not upgrading, you are missing out with a lot of new HomeKit features with iOS 11.2.  Some of the features that I have implemented:

- Sharing with my wife ... no issues, we both get notifications.
- Automations leaving (all people gone)/arriving (first person at home) at home work just fine.
- Automations when both are away, such as turning on lights based on time,  work just fine.

If you go for the upgrade, I would reboot Homebridge as well as your Apple TV, after all the upgrades are complete.  Then go in and explore the new features with HomeKit.

I know this did not answer your question but it seems like you are doing a lot of work to make iOS 10.3.3 work.

**Comment by ayanm00** [comment]
Yeah, I like the new features, but I'm wary of updating because I barely get through the day on iPhone 7 with 10.3.3 (I'm never near a plug), and I've heard that 11 has far worse battery life. I'm waiting for 11.3 so I can update, clean install, and restore all my messages.

**Comment by DMBlakeley** [comment]
There are lots of things that impact iPhone battery life from screen brightness to background activity.  Apple has a very good article at https://www.apple.com/batteries/maximizing-performance/.  Not mentioned, doing a restart often helps.

**Comment by ayanm00** [comment]
I'll look into that. Thanks for the article! In the meantime is there any way to get what I'm looking for? Somehow updating the state change of all instances of homebridge when one instance is updated?
